keyboards/handwired/steamvan/rev1/rev1.h

```diff
@@ -0,0 +1,98 @@
+// numbers of the keys on the PCB.
+
+#define LAYOUT_standard( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, \
+    K30, K31, K33,           K34, K36,           K38, K39, K3B  \
+) \
+{ \
+    { K00,   K01,   K02,   K03,  K04,   K05,   K06,   K07,   K08,   K09,   K0A,   K0B }, \
+    { K10,   K11,   K12,   K13,  K14,   K15,   K16,   K17,   K18,   K19,   K1A,   K1B }, \
+    { K20,   K21,   K22,   K23,  K24,   K25,   K26,   K27,   K28,   K29,   K2A,   K2B }, \
+    { K30,   K31,   KC_NO, K33,  K34,   KC_NO, K36,   KC_NO, K38,   K39,   KC_NO, K3B }  \
+}
+
+#define LAYOUT_arrow( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, \
+    K30, K31, K33,      K34, K36,      K38, K39, K3A, K3B  \
+) \
+{ \
+    { K00,   K01,   K02,   K03,  K04,   K05,   K06,   K07,   K08,   K09,   K0A,   K0B }, \
+    { K10,   K11,   K12,   K13,  K14,   K15,   K16,   K17,   K18,   K19,   K1A,   K1B }, \
+    { K20,   K21,   K22,   K23,  K24,   K25,   K26,   K27,   K28,   K29,   K2A,   K2B }, \
+    { K30,   K31,   KC_NO, K33,  K34,   KC_NO, K36,   KC_NO, K38,   K39,   K3A,   K3B }  \
+}
+
+#define LAYOUT_command( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, \
+    K30, K31, K32, K33,      K34, K36,           K38, K39, K3B  \
+) \
+{ \
+    { K00,   K01,   K02,   K03,  K04,   K05,   K06,   K07,   K08,   K09,   K0A,   K0B }, \
+    { K10,   K11,   K12,   K13,  K14,   K15,   K16,   K17,   K18,   K19,   K1A,   K1B }, \
+    { K20,   K21,   K22,   K23,  K24,   K25,   K26,   K27,   K28,   K29,   K2A,   K2B }, \
+    { K30,   K31,   K32,   K33,  K34,   KC_NO, K36,   KC_NO, K38,   K39,   KC_NO,   K3B }  \
+}
+
+#define LAYOUT_arrow_command( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, \
+    K30, K31, K32, K33,      K34, K36,      K38, K39, K3A, K3B  \
+) \
+{ \
+    { K00,   K01,   K02,   K03,  K04,   K05,   K06,   K07,   K08,   K09,   K0A,   K0B }, \
+    { K10,   K11,   K12,   K13,  K14,   K15,   K16,   K17,   K18,   K19,   K1A,   K1B }, \
+    { K20,   K21,   K22,   K23,  K24,   K25,   K26,   K27,   K28,   K29,   K2A,   K2B }, \
+    { K30,   K31,   K32,   K33,  K34,   KC_NO, K36,   KC_NO, K38,   K39,   K3A,   K3B }  \
+}
+
+// Backlighting
+typedef union {
+    uint8_t raw;
+    struct {
+        bool    enable :1;
+        bool    breathing : 1;
+        uint8_t level  :6;
+    };
+} backlight_levels_config_t;
```

Comment:
Just a note here - the core backlight config struct is now identical to this one, so redefining it is not necessary anymore in boards with custom backlight. I'm guessing this is all copypasted code though, and @zvecr's ARM backlight PR or followups will wash all of it away anyhow.

Reply:
Good to know, thanks @fauxpark! Should I remove it from the PR, or wait for the followup ARM backlight PRs?

Reply:
I'd say remove it now.

Reply:
Will do - addressed in commit 020ead637f226b77ca071f50f6fe6fcf1a815459